src/features/home/pages/Home.css

```
@@ -1,263 +1,272 @@
 
 /* h1 태그 스타일 */
 .main-content h1 {
-    font-size: clamp(1.5rem, 5vw, 4rem); /* 글자 크기 (rem 단위는 사용자 설정에 따라 조절됨) */
-    font-weight: 800; /* 글자 굵기  */
-    margin: 0; /* 위아래 간격 초기화 */
+  font-size: clamp(
+    1.5rem,
+    5vw,
+    4rem
+  ); /* 글자 크기 (rem 단위는 사용자 설정에 따라 조절됨) */
+  font-weight: 800; /* 글자 굵기  */
```

Stylelint (17.4.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)